pkgs/kde/third-party/krohnkite/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  kwin,
+  kpackage,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "krohnkite";
+  version = "0.9.7";
+
+  src = fetchurl {
+    url = "https://github.com/Ben9986/krohnkite/releases/download/${finalAttrs.version}/krohnkite-${finalAttrs.version}.kwinscript";
+    hash = "sha256-znXU3dDwR8CNAeZy0TJDk1R/qkUvhilmsLU11Y8kQOE=";
+  };
+
+  phases = [ "installPhase" ];
```

Comment:
Don't set phases explicitly, it just makes things annoying for overriding later.

Reply:
Without this it tries to unpack the `.kwinscript` file and fails, what would be the correct way to do it? thanks

Reply:
`dontUnpack = true`, but again, building from source is preferable.

Reply:
thanks :)